workflow/util/pod_name_test.go

```diff
@@ -34,6 +38,16 @@ func TestPodName(t *testing.T) {
 
 	assert.Equal(t, maxK8sResourceNameLength-k8sNamingHashLength-1, len(actual))
 
-	name = PodName(longWfName, nodeName, longTemplateName, nodeID, GetPodNameVersion())
+	name = PodName(longWfName, nodeName, longTemplateName, nodeID, PodNameV1)
 	assert.Equal(t, nodeID, name)
+
+	///////////////////////////////////////////////////////////////////////////////////////////
+	// POD_NAMES v2:
+	///////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Split into two tests instead of having this comment?

Reply:
Good idea